app/scripts/modules/core/src/pipeline/config/templates/configurePipelineTemplateModal.controller.ts

```diff
@@ -97,6 +109,14 @@ export class ConfigurePipelineTemplateModalController implements IController {
   }
 
   public buildConfig(): IPipelineTemplateConfig {
+    let inheritConfig = [];
+    if (this.state.inheritTemplateParameters) {
+      inheritConfig.push('parameters');
+    } else if (this.state.inheritTemplateExpectedArtifacts) {
```

Comment:
Shouldn't these be separate `if` blocks?

Reply:
Wasn't a fan of the conditional blocks anyway, cleaned it up